### PR TITLE
TurnTurn

### PR DESCRIPTION
**Pull request by JeffreySarnoff**

### CODE REVIEW

**Comment by yakir12** [comment]
fixes #8

File: test/runtests.jl [hunk 1]
```diff
@@ -2,11 +2,10 @@ using UnitfulAngles
 using Unitful
 using Base.Test
 
-units = (u"turn", u"halfTurn", u"quadrant", u"sextant", u"octant", u"clockPosition", u"hourAngle", u"compassPoint", u"hexacontade", u"brad", u"°", u"grad", u"arcminute", u"arcsecond", u"rad", u"diameterPart")
-quantities = (1, 2, 4, 6, 8, 12, 24, 32, 60, 256, 360, 400, 21600, 1296000, 2π, 120π)
+units = (u"turnturn", u"turn", u"halfTurn", u"quadrant", u"sextant", u"octant", u"clockPosition", u"hourAngle", u"compassPoint", u"hexacontade", u"brad", u"°", u"grad", u"arcminute", u"arcsecond", u"rad", u"diameterPart")
+quantities = (1, 2, 4, 8, 12, 16, 24, 48, 64, 120, 512, 720, 800, 43200, 2592000, 4π, 240π)
```
Comment (waldyrious) [inline test/runtests.jl:6]: What is the purpose of the change in these numbers?

**Comment by waldyrious** [comment]
Wouldn't it be more consistent to follow the pattern HalfTurn → Turn → DoubleTurn? TurnTurn strikes me as an odd name (but it might be due to English not being my first language).

File: test/runtests.jl (hunk 1 above)
Comment (JeffreySarnoff) [reply test/runtests.jl:6]: I wanted to include turnturn in the units testing.  Doing so adds one more factor of 2 into the way you designed the test.

**Comment by JeffreySarnoff** [comment]
I had spoken of it as DoubleTurn initially.  I changed to TurnTurn for two reasons: (b) DoubleTurn is technically hard to defend -- the 4pi rotation to return to a starting configuration is more primary than the 2pi revolution looking along a circumference.  TurnTurn is not a doubled Turn .. Turn is a halved TurnTurn. (b) It fits better with the length of the other names

File: test/runtests.jl (hunk 1 above)
Comment (JeffreySarnoff) [reply test/runtests.jl:6]: I don't know what I mistyped -- the test worked for me ... so I copied improperly

Comment (yakir12) [reply test/runtests.jl:6]: Sorry, but wouldn't it be better to just put a `0.5` first in `quantities`?

**Comment by yakir12** [comment]
(I agree with @waldyrious, `DoubleTurn` is a better name than `TurnTurn`.) Sorry, you answered before me. I don't understand your first point. Could you explain?

**Comment by JeffreySarnoff** [comment]
With respect the measurement of elemental mass, A Hydrogen Atom is more fundamental than a Helium Atom and would not make sense call Hydrogen's mass 1/2 just to allow Helium's mass to be called 1.
That is the situation with 0 --> 2pi --> 4pi==0  and 0--> 2pi as 0

**Comment by JeffreySarnoff** [comment]
I do not know what happened to the test -- call it DoubleTurn and restore the test.  I am fine with it, just busy now.

**Comment by yakir12** [comment]
Ah, I understand you now. 
While there are concrete examples where a double turn represents something that is both real and more fundamental than a turn, for most users it would just make more sense to call it a double turn. Keep in mind that we are already in "meta-level number one": we moved from the familiar and loved π to τ (i.e. in respect to the old argument on which of these should be used in math etc). To move one more level up again would be *bunkers*! 😃 

No rush with the PR, change to DoubleTurn and see my comments on the test. 
Thanks for this!

**Comment by waldyrious** [comment]
I would agree with @yakir12 here -- in some contexts 4π is more fundamental, in others π/2 is more fundamental, and it's impossible to pick a standard that will fit to all cases. The argument for clarity, then, becomes harder to counteract.

While we're at it, and considering that in English the only [specially-named fractions](https://en.wikipedia.org/wiki/Fraction_(mathematics)#Vocabulary) are "half" and "quarter" (the others are ordinals like "third", "fifth", "tenth", etc.) perhaps it would make sense to use QuarterTurn as an alias for Quadrant.

> See my comments on the test.

I don't see anything there -- did you perhaps forget to submit?

**Comment by waldyrious** [comment]
Possibly relevant: https://en.wikipedia.org/wiki/Spat_(unit)

**Comment by yakir12** [comment]
I wrote [here](https://github.com/yakir12/UnitfulAngles.jl/pull/9/files/129cea76d2afc7302b8ca0aacddc5ef210a7911e) this:

> Sorry, but wouldn't it be better to just put a `0.5` first in `quantities`?

**Comment by yakir12** [comment]
Yea, there are no 3D angles here..

**Comment by JeffreySarnoff** [comment]
+1 for QuarterTurn

**Comment by yakir12** [comment]
Feel free to add all the aliases you like. The overhead should be nil.

**Comment by JeffreySarnoff** [comment]
with apology, I have lost some context here -- closing
